Description: disable coverage report
 Without this patch, coverage reports are disabled. This currently fails under
 Ubuntu Zesty for unknown reasons. See also:
Author: Mathias Ertl <mati@jabber.at>
Bug: https://github.com/processone/fast_xml/issues/21
--- a/rebar.config
+++ b/rebar.config
@@ -34,7 +34,7 @@
 
 {clean_files, ["c_src/fxml.gcda", "c_src/fxml.gcno", "c_src/fxml_stream.gcda", "c_src/fxml_stream.gcno"]}.
 
-{cover_enabled, true}.
+{cover_enabled, false}.
 {cover_export_enabled, true}.
 
 {xref_checks, [undefined_function_calls, undefined_functions, deprecated_function_calls, deprecated_functions]}.
